3_visualize/src/plot_data.R

```diff
@@ -0,0 +1,45 @@
+plot_data <- function(plot_out, eval_data){
+  
+  # Create a plot
+  png(file = file.path(plot_out), width = 8, height = 10, res = 200, units = 'in')
```

Comment:
You don't need to use `file.path()` here anymore. It is used to take two strings and convert them into a path (e.g. `file.path("src", "myfile.R")` will return `"src/myfile.R"`). `plot_out` is already a filepath, so you can just use that alone.